lua/minigames/engine/sh_convars.lua

```diff
@@ -143,9 +150,62 @@ else
 			local mg = mgs[i]
 			local cvd = mg.conVarData
 
-			if not istable(cvd) or table.Count(cvd) < 1 then continue end
+			local size = 20
+
+			if not istable(cvd) or table.Count(cvd) < 1 then
+				size = 40
+			else
+				for _, data in pairs(cvd) do
+					if data.slider then
+						size = size + 25
+					end
+
+					if data.checkbox then
+						size = size + 20
+					end
+
+					if data.combobox then
+						size = size + 30
+
+						if data.desc then
+							size = size + 13
+						end
+					end
+
+					if data.label then
+						size = size + 13
+					end
+				end
+			end
+
+			clp = vgui.Create("DCollapsibleCategory", pnl)
+			clp:SetSize(390, size)
+			clp:SetExpanded(b and 1 or 0)
+			clp:SetLabel(LANG.TryTranslation("ttt2_minigames_" .. mg.name .. "_name"))
 
-			local size = 0
+			b = false
+
+			lst = vgui.Create("DPanelList", clp)
+			lst:SetPos(5, 25)
+			lst:SetSize(390, size)
+			lst:SetSpacing(5)
+
+			lst:AddItem(xlib.makecheckbox{
```

Comment:
Why not using the already included auto cvar system? https://github.com/TTT-2/ttt2mg#dynamical-ulx-convar-support, e.g. https://github.com/TTT-2/ttt2mg/blob/385dfde1571965880076e0c3cd16da231d3b5b97/lua/minigames/minigames/sh_bubblehead_minigame.lua#L8
Or expanding the list with default entries here: https://github.com/TTT-2/ttt2mg/pull/4/files#diff-fd97296f5e50bccad863dde7d14e8c84R151
Anyways, I didn't checked, but aren't minigames randomness and `enabled` already togglable with ULX?

Reply:
They were not auto-generated already